app/controllers/hosts_controller.rb

```diff
@@ -118,6 +118,7 @@ def update
     forward_url_options
     Taxonomy.no_taxonomy_scope do
       attributes = @host.apply_inherited_attributes(host_params)
+      attributes = @host.apply_interfaces_compute_attributes(attributes)
```

Comment:
Do we need this in APIv2 as well?

Reply:
Not sure about this. Maybe someone else can provide more insight ?

Reply:
We do. :-)